src/app/new-char/page.tsx

```diff
@@ -30,12 +35,19 @@ const characterCreation = () => {
       if (players === undefined)
         return alert('Erro 500 algo de errado aconteceu...');
       setPlayers([...players, newCharacter]);
+      alert(`${newCharacter.name} has been added to the server.`);
+      setButtonDisabled(false);
     };
   };
 
   return (
     <Wrapper>
-      <Form action={sendToServer}>
+      <Form
+        action={(formData) => {
+          handleCharacterCreation();
+          sendToServer(formData);
+        }}
+      >
```

Comment:
Se as sugestões acima forem seguidas, ficará:
```suggestion
      <Form action={handleCharacterCreation}>
```